tmserver/api/mapobject.py

```diff
@@ -496,7 +496,7 @@ def add_segmentations(experiment_id, mapobject_type_id):
         well_pos_x, zplane, tpoint
     )
 
-    npz_file = base64.b64decode(data.get('npz_file'))
+    npz_file = base64.decodestring(data.get('npz_file'))
     pixels = np.load(BytesIO(npz_file))["segmentation"]
```

Comment:
Please see https://github.com/TissueMAPS/TmClient/pull/25/files#r167567991 -- same applies here, IMHO

Reply:
Agree - I reset the deprecated function call.